app/Http/Controllers/SignupsController.php

```diff
@@ -65,7 +65,7 @@ public function store(Request $request)
         $northstarId = getNorthstarId($request);
 
         // Check to see if the signup exists before creating one.
-        $signup = $this->signups->get($northstarId, $request['campaign_id'], $request['campaign_run_id']);
+        $signup = $this->signups->get($northstarId, $request['campaign_id']);
```

Comment:
@DFurnes - I think this line may what's causing some unexpected behavior in Gambit, where [we're not seeing new signups from yes responses to the TYF broadcast when a user has signed up for the campaign in a previous run](https://dosomething.slack.com/archives/C1V0M6RPE/p1544051436007200).

Reply:
Oh, shoot. You’re totally right. We can revert this PR this morning to resolve that problem.

I think our best course of action is to avoid removing runs from applications until we’ve run the script in Rogue, since otherwise we don’t have a reliable way to know whether or not a signup is new. (Or we could adjust this to find the latest run for _all_ signups on a given campaign ID, but I’m thinking that adds some unnecessary complexity and a chance we introduce other bugs.)

Reply:
I’ll update the order-of-operations issue to reflect this. cc: @chloealee @weerd